android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsAppOpenModule.java

```diff
@@ -119,7 +119,8 @@ public void onAdFailedToLoad(@NonNull LoadAdError loadAdError) {
   }
 
   @ReactMethod
-  public void appOpenShow(int requestId, ReadableMap showOptions, Promise promise) {
+  public void appOpenShow(
+      int requestId, String adUnitId, ReadableMap showOptions, Promise promise) {
```

Comment:
That's odd :thinking: - there is no action necessary, but if I understand correctly this is a spacing-only change? And I thought we were enforcing spacing/formatting with google-java-format via https://github.com/invertase/react-native-google-mobile-ads/blob/8bbc0ad742bb0a23b6fecdd718e701985d21086b/package.json#L71

So I wouldn't expect to see any changes like this unless lint demanded them. So either it was failing before and CI has a false-positive (so needs investigation) *or* google-java-format allows both? *or* something unknown.

Not a big deal at all but I try to use auto-formatting and auto-format-checks as much as possible so that diffs are small and no one argues about spacing (total waste of time in my opinion haha)